test/controllers/areas_controller_test.rb

```diff
@@ -1,7 +1,32 @@
 require "test_helper"
+require_relative '../support/controller/parse_response'
+require_relative '../support/controller/areas_params'
+require_relative '../support/controller/areas_assertions'
 
 class AreasControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
+  include ParseResponse
+  include AreasParams
+  include AreaAssertions
+
+  test 'get index is successful' do
+    get areas_path
+    assert_response :success
+    assert_equal Area.all.size, length_body, 'Index all areas'
+  end
```

Comment:
Es importante usar variables muy semánticas y buenas descripciones en los errores porque esta es nuestra documentación, así como esta muy bien testear qué data me entrega el endpoint, porque si llego a cometer algún error en un refactor el test no me va a aportar valor porque no se va a morir si la embarro cambiando una llave.

```suggestion
  test 'Valid index Areas' do
    get areas_path
    
    assert_response :success
    
    all_areas_size = Area.all.size
    all_areas_response = length_body
    assert_equal all_areas_size, all_areas_response, 'Describe mejor estos errores'
  end
```